### ISSUE TITLE
[Suggestion] Change the name 

### ISSUE THREAD

**Issue by phiberber** [moomooooo/postman#13]
**Description:** The name is already used by a company, it is a software to test REST requests. By that making it harder to find the cheat itself (happened with me some months ago).
**Solution**: Change the name to another one, you didn't explain the meaning behind the name postman but I think we _are_ able to find a better name that even fits it better;

**Comment by moomooooo** [comment]
Thank you phiberber, I got the name postman from a movie. I will not change the name, I don't intend on this being a big project, its something I started for myself, and I'm glad its catching on, but I don't plan on devving it very far in the future. The name has caught on quite a bit in the community, and changing that would hurt it quite a bit. I'd rather stick to this for as long as possible, continue to take feedback form others, and make people happy to find a client like it, rather than switch the name and lose a good portion of traction, just before moving on to other projects. I'll continue to work on this client for quite a while, but changing the name is a bit unnecessary for something like this. Also, there are ways I can make the client easier to find, I've done a lot through discord, gotten it added to different archives, the more popular the gets, the more people use it and advertise the name on servers, or through videos, and I can start a website with an actual domain to make the client easier to find.
